objcgen/objcgenerator.cs

```diff
@@ -562,6 +562,18 @@ void GenerateArgumentArray (string parameterName, string argumentName, Type t, r
 				implementation.WriteLine ("else");
 				implementation.Indent++;
 				implementation.WriteLine ($"mono_array_set ({pnameArr}, {typeName}, {pnameIdx}, {pnameRet}.{returnValue});");
+				implementation.Indent--;
```

Comment:
That's out of code change. Is that fixing the current indentation issues ? (noticed one earlier today)